compiler/ilgen/IlBuilder.cpp

```diff
@@ -539,6 +540,11 @@ IlBuilder::AppendBuilder(TR::IlBuilder *builder)
    cfg()->addEdge(builder->getExit(), _currentBlock);
    }
 
+void
+IlBuilder::appendExceptionBuilder(TR::IlBuilder *builder, TR::Block *blockThrowsException)
```

Comment:
Same comment about TR::Block parameter. You need to figure out a different way to do it.